views/components/Config/ConfigCardAvatar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Avatar from '@material-ui/core/Avatar'
+
+import { colors } from '../../../constants'
+
+const ConfigCardAvatar = ({ ownsConfig, configOwner, userAvatar }) => {
+  if (!ownsConfig) {
+    const { icon, uid } = configOwner
+    if (!!icon) {
+      return (
+        <Avatar
+          src={icon}
+          alt=""
+          style={{
+            width: 60,
+            height: 60,
+            backgroundColor: colors.columbia_blue,
+          }}
+        />
+      )
+    } else {
+      return (
+        <Avatar
+          style={{
+            width: 60,
+            height: 60,
+            backgroundColor: colors.columbia_blue,
+          }}
+        >
+          {uid[0]}
+        </Avatar>
+      )
+    }
+  } else {
+    return userAvatar
+  }
```

Comment:
this looks good but I think we can simplify the logic slightly for readability. What do you think about the below suggestion?
```suggestion
  if (!!ownsConfig) {
    return userAvatar
  }

  const { icon, uid } = configOwner
  
  return !!icon
   ? <Avatar src={icon} alt="" style={AVATAR_STYLE} />
   : <Avatar style={AVATAR_STYLE}>{uid[0]}</Avatar>
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/196/commits/159c5b43ba6f258f9ca42cfd988c9f29017e7d09